### PR TITLE
Bump to VuePress 2.0

### PR DESCRIPTION
**Pull request by Nofated095**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -18,6 +18,6 @@
   },
   "homepage": "https://github.com/LittleSkinCommspt/commspt-bot-manual#readme",
   "devDependencies": {
-    "vuepress": "^1.5.4"
+    "vuepress": "^2.0.0-beta.45"
```
Comment (Qubik65536) [inline package.json]: Is this a stable version? I don't think so.

Cubik65536 at #15

> Beta releases should NOT be used in production.

FLYEMOJ1 at #15

> > Beta releases should NOT be used in production.
>
> Agree with that.

jinzhijie at #15 

> 暂时不考虑升级到非稳定版本

**Review by Qubik65536 — DISMISSED** [review]
Request changes, or, I **REFUSE** these changes.

File: package.json (hunk 1 above)
Comment (gxres042) [reply package.json]: So what can we do? Update to VuePress v1.9.x?
Also we need to solve dependencies alert.

**Comment by Nofated095** [comment]
Production Review at **<https://ls-bot.9595095.xyz/>** Will Close After Merged
# [vuepress/vuepress-next 生产环境使用beta是否合适 #955](https://github.com/vuepress/vuepress-next/discussions/955)

**Comment by gxres042** [comment]
@Cubik65536 Remember that LittleSkin Manual said that "LittleSkin use the DEV version of Blessing Skin"?

I think we can do that now. Why not try it?

**Review by gxres042 — COMMENTED** [review]
But I have a require.

**If bump to VuePress 2 can resolve the 14 security alerts that appear now, I agree to bump to VuePress 2.**

**Comment by Qubik65536** [comment]
> @Cubik65536 Remember that LittleSkin Manual said that "LittleSkin use the DEV version of Blessing Skin"?
> 
> 
> 
> I think we can do that now. Why not try it?

I **GIVE UP** from talking about it. If you guys think it's the only thing important to do, then do it.

**Review by SerinaNya — APPROVED** [review]

**Comment by gxres042** [comment]
> > @Cubik65536 Remember that LittleSkin Manual said that "LittleSkin use the DEV version of Blessing Skin"?
> > I think we can do that now. Why not try it?
> 
> I **GIVE UP** from talking about it. If you guys think it's the only thing important to do, then do it.

**No the most important thing.**

I think it's necessary to get your options because you first request changes.

**I propose a require to him to do. Just need to resolve all the security alerts that appears now.** If he can't, maybe I will rebuild this bot manual with the latest **STABLE** VuePress.

Thanks for your reply. You can suggest any options to this. I will listen.

**Review by Qubik65536 — COMMENTED** [review]
``` plain
Remove change request
```

**Comment by Nofated095** [comment]
image
I deleted caches and rebuilt the VuePress.
Seems nothing was wrong expect ```source-map-resolve```. Seems it's necessary for VuePress to use the package. It's not an "unmet peer dependency" problem.

**Comment by Nofated095** [comment]
Now VuePress v2 still has some bugs that can be clearly perceived.
The commspt-bot-manual has a long title and VuePress can't hide the title correctly, especially on the mobile devices.
image

**Pull request merged**